plugins/plugin-docker/che-plugin-docker-client/src/test/java/org/eclipse/che/plugin/docker/client/params/BuildImageParamsTest.java

```diff
@@ -211,205 +231,63 @@ public void shouldThrowIllegalStateExceptionIfAddFileAfterSetRemote() {
 
     @Test
     public void repositoryParameterShouldEqualsNullIfItNotSet() {
-        buildImageParams.withTag(TAG)
```

Comment:
Then it is possible that second setting breaks everything or third or ... Should we test all these cases? I think no, then it doesn't make sense to set all possible fields in each test.